controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -129,6 +129,15 @@ public ResponseEntity<FullHealthDataAssessmentResponse> postFullHealthAssessment
 
       FullHealthDataAssessmentResponse response =
           objectMapper.readValue(responseAsString, FullHealthDataAssessmentResponse.class);
+      if (response.getStatus().equals("ERROR")) {
+        log.info(
+            "Response from condition processor returned error message: {}",
+            response.getErrorMessage());
+        throw new ClaimProcessingException(
+            claim.getClaimSubmissionId(),
+            HttpStatus.INTERNAL_SERVER_ERROR,
+            "Error in condition processor.");
+      }
       if (response.getEvidence() == null) {
         throw new ClaimProcessingException(
             claim.getClaimSubmissionId(), HttpStatus.INTERNAL_SERVER_ERROR, "No evidence found.");
```

Comment:
This error still does not make sense. If there is no evidence found I am assuming you are supposed to return empty lists. And I do not even think you would come here after the other changes in this PR. If there is a path here you should change the error messages.